compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/HttpFiltersDesugar.java

```diff
@@ -233,6 +234,11 @@ private BLangSimpleVariable addFilterContextCreation(BLangFunction resourceNode,
         return filterContextVar;
     }
 
+    private String extractResourceName(BLangFunction resourceNode) {
```

Comment:
Is this method supposed to extract the resource paths?
say we have a resource function such as `resource function get foo/bar/[int i]/[int j]() { }` what is the expected behavior?

Reply:
The objective of this function is to extract the resource name by removing the accessor part attached to it. The expected value for the given resource signature is `"foo$bar$*$*"` and this function returns that.

Reply:
Ack, I was confused with the term `resourceName` as resource functions doesn't have a name anymore. Shall we concatenate the text parts of [this](https://github.com/ballerina-platform/ballerina-lang/blob/70ed2d553613073ed0bfd4ace1fcc20bd5f56755/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangResourceFunction.java#L33) instead of removing from the generated string.